client/src/components/Invite/Invite.scss

```diff
@@ -0,0 +1,25 @@
+@import 'styles/variables';
+@import 'styles/helpers';
+
+.Invite {
+  h2 {
+    margin-bottom: $spacing-small;
+  }
+  &__LinkBlock {
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: $spacing-small;
+    align-items: center;
+    [type='text'] {
+      flex-grow: 3;
+      margin-right: $spacing-small;
+      background-color: $color-grey;
+      border-radius: $radius-regular;
+      padding: $spacing-small;
+      border: none;
+    }
+    .Button {
```

Comment:
It's better to apply a class in the component where you're instantiating `<Button />`. It prevents having side effects when surcharging original classes of the component.